transactor/src/main/scala/io/mediachain/copycat/Client.scala

```diff
@@ -121,21 +122,37 @@ class Client(client: CopycatClient) extends JournalClient {
     stateListeners.foreach(_.onStateChange(Disconnected))
   }
   
+  private def recover() {
+    val cf = client.recover()
+    recoveryExecutor.submit(new Runnable {
+      def run {
+        try {
+          cf.join()
+          logger.info("Copycat session recovered")
+        } catch {
+          case e: InterruptedException => ()
```

Comment:
yeah, i got it. 
It doesn't matter all that much with the interrupt exception, the worst thing that can happen is some errant logging at shutdown.